docs/interpreter/cassandra.md

```diff
@@ -134,49 +138,53 @@ Insert into users(login,name) vAlues('hsue','Helen SUE');
 ```
 
 The complete list of all CQL statements and versions can be found below:
```

Comment:
Here is the same with the line number 203. In the below table, there are unpurposely added spaces after `<center>`, `</tr>` tag. Those spaces will make useless `<br>` tag in the web page.
![screen shot 2016-02-05 at 11 27 29 am](https://cloud.githubusercontent.com/assets/10060731/12836366/8e3f64ec-cbfb-11e5-89d1-4c9a824e151f.png)